src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
+
+export const appConfig: ApplicationConfig = {
```

Comment:
suggestion

          export const appConfig: ApplicationConfig = {

          Explantion: The export statement is fine, but consider adding type annotations if you're adding more providers in the future for clarity.